src/components/sandbox-menu/sandbox-menu.tsx

```diff
@@ -48,13 +49,26 @@ export interface SandboxArgs {
 }
 
 function saveConfig(baseUrl: string, config: Record<string, unknown>) {
+  ConsoleLogger.debug(`sandbox-menu: saveConfig: ${baseUrl}`, config);
+
+  const toBeCleared = Object.entries(config)
```

Comment:
@TheRedDaemon This is the new trashing behavior, e.g. if a key is set to undefined, then it is removed from the user config. I see I made a mistake because I didn't prepend `baseUrl` yet, as you do on line 62.

Reply:
Ok, then please fix this.

Reply:
Looks good